### ISSUE TITLE
Does not compile.

### ISSUE THREAD

**Issue by Zibri** [pytries/datrie#39]
while pip3 install datrie   works,
# pip install git+https://github.com/pytries/datrie

    running build_ext
    building 'datrie' extension
    creating build/temp.linux-x86_64-2.7/src
    cc -O2 -fPIC -Wimplicit -O2 -fPIC -Wimplicit -Ilibdatrie -I/usr/lib/pypy/include -c src/datrie.c -o build/temp.linux-x86_64-2.7/src/datrie.o
    src/datrie.c:4:20: fatal error: Python.h: No such file or directory
     #include "Python.h"
                        ^
    compilation terminated.
    error: command 'cc' failed with exit status 1

please fix that!!!

**Comment by superbobry** [comment]
Does it compile under CPython?

**Comment by jerryasher** [comment]
On Ubuntu 16.04, I was having problems installing it into Python 3.6. 

I was able to install it under the Python 3.5 that I think Ubuntu shipped with, but it would break similar to the above when I installed it after installing Python 3.6.3.

Amongst my various fixes were to:

+ python3.6 -m pip install pip --upgrade
+ apt install python3.6-dev
+ pip3.6 install datrie

After that it seems to have installed just fine.
